lib/irq.c

```diff
@@ -322,6 +344,13 @@ device_set_irqs_validate(vfu_ctx_t *vfu_ctx, vfu_msg_t *msg)
         line = __LINE__;
         goto invalid;
     }
+    //count must be 0 or 1 for ERR/REQ
+    if (((irq_set->index == VFIO_PCI_ERR_IRQ_INDEX) ||
+         (irq_set->index == VFIO_PCI_REQ_IRQ_INDEX)) &&
```

Comment:
thanks for adding this, but I think you can definitely add a pytest for this case, let us know if it's not obvious?

Reply:
ok, I will add it in test_device_set_irqs.py.